db/openshift.deploy.yml

```diff
@@ -241,7 +245,7 @@ objects:
         - ReadWriteOnce
       resources:
         requests:
-          storage: "${PVC_SIZE}"
+          storage: "${DB_BACKUP_PVC_SIZE}"
```

Comment:
maybe we can removed the old '"PVC_SIZE" variable defined on line 93?

Reply:
Right, forgot removing this one, Thanks!